Resources/public/Admin.js

```diff
@@ -312,7 +312,7 @@ var Admin = {
         Admin.log('[core|setup_collection_counter] setup collection counter', subject);
 
         // Count and save element of each collection
-        var highestCounterRegexp = new RegExp('_([0-9])+$');
+        var highestCounterRegexp = new RegExp('_([0-9]+)$');
```

Comment:
Sorry i was focused on my dummy collection... but the original doesn't work neither :-/.
What about ``_([0-9]+)[^0-9]*$`` if you want the last number in the id string ?